pinot-segment-local/src/test/java/org/apache/pinot/segment/local/aggregator/DistinctCountCPCSketchValueAggregatorTest.java

```diff
@@ -34,19 +34,18 @@ public class DistinctCountCPCSketchValueAggregatorTest {
   @Test
   public void initialShouldCreateSingleItemSketch() {
     DistinctCountCPCSketchValueAggregator agg = new DistinctCountCPCSketchValueAggregator(Collections.emptyList());
-    assertEquals(agg.getInitialAggregatedValue("hello world").getEstimate(), 1.0);
+    assertEquals(toSketch(agg.getInitialAggregatedValue("hello world")).getEstimate(), 1.0);
   }
 
   @Test
   public void initialShouldParseASketch() {
-    CpcSketch input = new CpcSketch();
-    IntStream.range(0, 1000).forEach(input::update);
+    CpcSketch input = new CpcSketch(12);
+    IntStream.range(0, 100).forEach(input::update);
```

Comment:
What is the impact of this change? decreasing test initialization time?

Reply:
The sketch estimate was off by one on the previous configuration.  Rather than introducing a delta for the variable error, I decreased the number of inputs making the test more deterministic.